diff --git a/node_modules/react-native/Libraries/Core/ErrorUtils.web.js b/node_modules/react-native/Libraries/Core/ErrorUtils.web.js
new file mode 100644
index 0000000..a76d229
--- /dev/null
+++ b/node_modules/react-native/Libraries/Core/ErrorUtils.web.js
@@ -0,0 +1 @@
+﻿export default { setGlobalHandler: () => {}, getGlobalHandler: () => () => {}, reportError: () => {} };
diff --git a/node_modules/react-native/Libraries/Core/InitializeCore.web.js b/node_modules/react-native/Libraries/Core/InitializeCore.web.js
new file mode 100644
index 0000000..b511c97
--- /dev/null
+++ b/node_modules/react-native/Libraries/Core/InitializeCore.web.js
@@ -0,0 +1 @@
+﻿export default {};
